### ISSUE TITLE
How to use a different dataset

### ISSUE THREAD

**Issue by hipoglucido** [facebook/NAMAS#12]
Hello, I am interested into implementing your project. I have +7M (headline,article) pairs locally (stored in a SQL). I would like to know which steps do I have to follow to use it instead of the gigaword dataset. I mean, I would like to know an example of the input format of the trainning model so I could develop a python script to convert  my dataset into that format.

Thanks a lot

**Comment by abhishekkrthakur** [comment]
@hipoglucido did u manage?

**Comment by hipoglucido** [comment]
Nope, I used [this other project](https://github.com/udibr/headlines) instead

**Comment by abhishekkrthakur** [comment]
Yeah. But it isn't an attention based model if i understand correctly.

**Comment by falcondai** [comment]
it is certainly doable. the model trains on pairs of word sequences (first sentence, headline). you can use a standard tokenizer from NLTK to tokenize your text.

**Comment by arehmanAzam** [comment]
@falcondai  could you please share the example of this tokenization and its usage with NAMAS.

**Comment by falcondai** [comment]
@arehmanAzam you can consult my fork: https://github.com/falcondai/NAMAS/blob/master/dataset/process_agiga.py#L76

**Comment by arehmanAzam** [comment]
Thanks @falcondai
